wrappers/jsprit.rb

```diff
@@ -389,7 +389,7 @@ def parse_output(path, iterations, fleet, vrp)
       if solution
         {
           cost: Float(solution.at_xpath('cost').content),
-          costs: Models::Costs.new({}), # TODO: fulfill with solution costs
+          cost_details: Models::CostDetails.new({}), # TODO: fulfill with solution costs
```

Comment:
Why do we still hold on to this file. I am ~100% sure it doesn't work :) If we delete it git history will take good care of it for us. :D

Reply:
Maybe another ticket.